### PR TITLE
Fix some loggers which are identifying as the wrong class.

### PR DESCRIPTION
**Pull request by deadmoose**

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
:+1:
Good case for static analysers BTW

**Comment by daniel-beck** [comment]
Did you confirm that this wasn't intentional via the commit history? I can imagine some of these being intentional after refactoring and not wanting to break users' existing log recorders.

File: core/src/main/java/hudson/model/ItemGroupMixIn.java [hunk 1]
```diff
@@ -104,11 +104,11 @@ public boolean accept(File child) {
...
                 if (item == null) {
-                    XmlFile xmlFile = Items.getConfigFile( subdir );
+                    XmlFile xmlFile = Items.getConfigFile(subdir);
                     if (xmlFile.exists()) {
-                        item = (V) Items.load( parent, subdir );
-                    }else{
-                        Logger.getLogger( ItemGroupMixIn.class.getName() ).log( Level.WARNING, "could not find file " + xmlFile.getFile());
+                        item = (V) Items.load(parent, subdir);
+                    } else {
+                        Logger.getLogger(ItemGroupMixIn.class.getName()).log(Level.WARNING, "could not find file " + xmlFile.getFile());
```
Comment (oleg-nenashev) [inline core/src/main/java/hudson/model/ItemGroupMixIn.java]: It's better to make the logger a static variable just to follow the common pattern

**Comment by oleg-nenashev** [comment]
Unfortunately, we will have a regression in corner cases. Loggers inheritance seems to be logically strange in the most of the proposed changes, so it is not an option. I would vote for review changes one-by-one and merge those ones where there's no logical dependencies between classes (everything excepting <code>PlainTextConsoleOutputStream</code>). The last case should be discussed.

@deadmoose 
An Jenkins JIRA issue with a list of incorrect loggers could be useful for the future reference in any case.

**Comment by KostyaSha** [comment]
:+1:  if it fixes all inconsistency in core. @daniel-beck i don't want debug code to know about additional loggers if i'm requesting Node logging, this just adds complexity for debugging.

File: core/src/main/java/hudson/model/ItemGroupMixIn.java [hunk 2]
```diff
@@ -104,11 +104,11 @@ public boolean accept(File child) {
                 // Try to retain the identity of an existing child object if we can.
                 V item = (V) parent.getItem(subdir.getName());
                 if (item == null) {
-                    XmlFile xmlFile = Items.getConfigFile( subdir );
+                    XmlFile xmlFile = Items.getConfigFile(subdir);
```
Comment (jglick) [inline core/src/main/java/hudson/model/ItemGroupMixIn.java]: Please avoid whitespace-only changes in PRs. This whole diff hunk appears to be gratuitous.

**Comment by oleg-nenashev** [comment]
@daniel-beck Do you agree with the merge?

**Comment by daniel-beck** [comment]
No opinion.

**Pull request merged**